model/project_ref.go

```diff
@@ -1983,18 +1983,21 @@ func GetProjectSettings(p *ProjectRef) (*ProjectSettings, error) {
 		return nil, errors.Wrapf(err, "finding subscription for project '%s'", p.Id)
 	}
 
-	hasGithubAppAuth, err := HasGithubAppAuth(p.Id)
-	if err != nil {
-		return nil, errors.Wrapf(err, "finding hadGithubApp for project '%s'", p.Id)
-	}
-
 	projectSettingsEvent := ProjectSettings{
 		ProjectRef:         *p,
 		GithubHooksEnabled: hasEvergreenAppInstalled,
 		Vars:               *projectVars,
 		Aliases:            projectAliases,
 		Subscriptions:      subscriptions,
-		HasGitHubApp:       hasGithubAppAuth,
+	}
+
+	githubAppID, err := GetGitHubAppID(p.Id)
+	if err != nil {
+		return nil, errors.Wrapf(err, "finding hadGithubApp for project '%s'", p.Id)
```

Comment:
`hadGithubApp` -> userfriendly `github app` or new method `GetGitHubAppID`

Reply:
true, thank you! fixed